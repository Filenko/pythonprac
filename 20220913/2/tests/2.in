[12, 1.1, 2, 16, 28, 11]
