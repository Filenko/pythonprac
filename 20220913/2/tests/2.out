1.1, 2, 11, 12, 16, 28
